tasks/Module 3/test_module_3.py

```diff
@@ -0,0 +1,23 @@
+"""Test all we can in Module 3."""
+import unittest
+import task_2
+
+
+class TestModule2(unittest.TestCase):
+    """TestCase  class."""
+
+    def test_my_split(self):
```

Comment:
I'd add a test case where your string is an equal separator. 
Also would like to see the test of your split vs original split to ensure results are the same.

I see you used unittest for the testing purposes, wonder if you want to give `pytest` a try? It's optional, I'll leave it up to you.